### ISSUE TITLE
Default logger path for new projects is wrong

### ISSUE THREAD

**Issue by rijkvanzanten** [directus/api#1178]
This default path:

https://github.com/directus/api/blob/f542f2f458825709dcb9e9ebe3ceb39e5af5350d/src/core/Directus/Util/Installation/stubs/config.stub#L11

will result in `Caught exception: Invalid Configuration for logger "path" variable.` every time a log is written. I think the `/../` in the path might be invalid. 

For the "default" project `_`, the log path is set to `'path' => __DIR__ . '/logs/app.log',` by default. This means that the logs are actually written _in_ the `config` folder, which is also not desirable...

We should change the default version to be `/logs/ <project_name >/`, and use `/logs/_/` as the path for the default project.

**Comment by rijkvanzanten** [comment]
It seems like

```php
[
  'path' => realpath(__DIR__ . '/../logs/')
]
```

at least fixes the config error

**Comment by WoLfulus** [comment]
realpath doesn't really make sense, since if the folder doesn't exists it will return false. the error raises when the setting isn't a directory or isnt writeable

does it exists?

**Comment by bjgajjar** [comment]
Hey @rijkvanzanten 

I am not able to replicate this issue, Below are the steps which I have performed.

1. Clone the `directus` build
2. Try to install the project from `APP`
3. Add the info log 
4. The info log generated in `logs` directory 

So basically, I got the logs **which is not generated in `config` folder.** but it was generated in the `api/logs` directory

I tried to connect with @hemratna as well as @WoLfulus regarding this issue. They are also not able to reproduce this issue.

Can you please help me how to figure it out?

**Comment by rijkvanzanten** [comment]
Yeah on my stack the config folder is on a shared drive, so /../ is unavailable for the project. Having https://github.com/directus/api/issues/1179 will fix it for our use case

**Comment by ersinpw** [comment]
@rijkvanzanten I am not able to setup Directus using Nginx. 

This is the error (500) I get:
> FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Directus\Logger\Exception\InvalidLoggerConfigurationException: Invalid Configuration for logger "path" variable. in www/src/core/Directus/Application/CoreServicesProvider.php:143

Which is similar you wrote above. 

My nginx settings are similar like:
https://docs.directus.io/advanced/server-setup.html#nginx

Furthermore, I downloaded Directus repository via GIT
`git clone https://github.com/directus/directus.git`

I tried to change the realpath, but that didn't work for me.
